docker-compose-local.yaml

```
@@ -1,21 +1,46 @@
+    ports:
+      - "8081:8081"
+    volumes:
+      - ./catalog/internal/catalog/testdata:/testdata
+
   model-registry:
     build:
       context: .
       dockerfile: Dockerfile
-    command: ["proxy", "--hostname", "0.0.0.0", "--datastore-type", "embedmd", "--embedmd-database-dsn", "root:demo@tcp(mysql:3306)/model_registry?charset=utf8mb4"]
+    entrypoint: ["/bin/sh"]
+    command: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)